python/triqs_dft_tools/converters/wannier90.py

```diff
@@ -61,7 +61,8 @@ class Wannier90Converter(ConverterTools):
 
     def __init__(self, seedname, hdf_filename=None, dft_subgrp='dft_input',
                  symmcorr_subgrp='dft_symmcorr_input', misc_subgrp='dft_misc_input',
-                 repacking=False, rot_mat_type='hloc_diag', bloch_basis=False, add_lambda=None):
+                 repacking=False, rot_mat_type='hloc_diag', bloch_basis=False, add_lambda=None,
+                 w90zero=2.e-6):
```

Comment:
Could you please add `w90zero` as parameter in the docstring below line 89. Then the reference in the doc is correctly generated. Thanks!

Reply:
Sure, sorry that I forgot. It's added in the latest version.